sdk/search/azure-search/samples/async_samples/sample_facet_query_async.py

```diff
@@ -0,0 +1,54 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_filter_query_async.py
```

Comment:
Should this match the name of the file?

Reply:
Thank you, fixed in https://github.com/Azure/azure-sdk-for-python/pull/10207/commits/81e88769a8b010142e68e55ba187e319e9b49947